index.rst

```diff
@@ -44,14 +44,639 @@
+The front-end is web based and supports natural zooming and panning, including zooming to full pixel resolution, as well as the ability to display pixel coordinates and raw pixel value by moving the mouse over the image. 
+It supports a simple but extensible set of image processing algorithms which have been found to be useful for viewing images during camera testing.
+
+The tool uses the `International Image Interchange Framework <https://iiif.io/>`_ (IIIF) with the front-end based on the open source `OpenSeadragon <https://openseadragon.github.io/>`_ framework, and the backend based on the open source `Cantaloupe <https://cantaloupe-project.github.io/>`_ product, modified to be able to read images from  FITS files and directly from the DAQ.
+The tool is currently deployed at SLAC for camera construction, and on the summit in Chile for Auxtel and ComCam.
+
+
+Engineering Facilities Database and Large File Annex
+'''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The `Engineering Facilities Database (EFD) <https://sqr-034.lsst.io/#introduction>`_ records all data sent by components over the DDS control network.
+This content essentially tracks the observatory state as a function of time and is very useful in diagnosing issues and understanding (both desired and undesired) operational behaviours.
+The database is best queried using the `EFD client <https://efd-client.lsst.io/>`_ from the `Nublado (Jupyter Notebook) Interface`_ (or any python-based method/script) when making custom plots.
+Instructions on how to access the EFD, and specifically the other instances of the data, are found in `SQR-034 <https://sqr-034.lsst.io/#efd-deployments>`_.
+However, the `Chronograf`_ graphical front-end offers a nice solution for building simple plots (dashboards).
+
+The `Large File Annex (LFA) <https://tstn-029.lsst.io/>`_ contains files over ~1 MB that are accessible both from the summit and the `RSP`_.
+When a file is published to the LFA its presence (and location of the file) is published via SAL/DDS and therefore the location of LFA files can be found via an EFD query.
+It is anticipated that artifacts generated from automated on-the-fly analyses will be stored in this area.
+An example of this would be the `Papermill Executed Parameterized Notebooks`_.
+
+
+Chronograf
+''''''''''
+
+Chronograf is a tool that provides a user-friendly graphical interface to each `deployed instance of the EFD <https://sqr-034.lsst.io/#efd-deployments>`_.
+During operations, observers will use the `summit instance <https://chronograf-summit-efd.lsst.codes/>`_ (VPN required).
+It is particularly useful for quick and simple analysis as well as for creating dashboards to show a specific information about the status of the observatory; especially when a similar functionality is not available in LOVE.
+
+It is not well suited for complex queries or figures and previous queries/plots are not easily replicated.
```

Comment:
I think this paragraph needs some rewording.. 

Here is a suggestion:

> Nevertheless, Chronograf is not particularly well suited for complex queries and figures, especially those that require some additional logic to be meaningful.
> For instance, if a Controllable SAL Component (CSC) crashes, a view displaying its summary state will continue to show the last value in the EFD, which is likely no longer relevant.
> To derive relevant information, it is necessary to combine it with some additional data, such as the time since the last published heartbeat. 
> Furthermore, Chronograf views (queries, plots, dashboards, etc) are not easily replicated, and can vary significantly for different users, depending on their local configurations.
> For these reason (and many others), it is not an appropriate substitute for a true monitoring system, such as what is provided by the `LOVE`_ interface.